storage/postgres-kysely/.env.example

```diff
@@ -0,0 +1,6 @@
+POSTGRES_URL=
+POSTGRES_URL_NON_POOLING=
+POSTGRES_USER=
+POSTGRES_HOST=
+POSTGRES_PASSWORD=
+POSTGRES_DATABASE=
```

Comment:
Should this variables also be part of the deploy button?

Reply:
no because we are using a special param for adding Vercel Postgres